build.gradle

```diff
@@ -410,3 +410,13 @@ task deployAll{
     dependsOn "server:deploy"
     dependsOn "android:deploy"
 }
+
+task resolveDependencies {
```

Comment:
Incorrect style - space before brace, see the [guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md#style-guidelines).

Reply:
Of course, sorry for not noticing first. Is it better now?